### DISCUSSION TITLE
Portal Feature: Add Deployment "Changes Summary" Info to "Environment History"

### DISCUSSION THREAD

**Discussion by hfloyd** [umbraco/Umbraco.Cloud.Issues#1013] [Ideas]
Every time you do a Deployment, you are forced to type something in a "Changes Summary" box. This information is then not available for viewing anywhere (except the most recent on the page which lists all your projects), which is pointless. 

My suggestion is to insert those Deployment notes in the Environment History page, above the specific changes in question. This would make the exercise of filling in the Changes Summary useful.

Example:
![image](https://user-images.githubusercontent.com/1252593/66505941-57c67d80-ea9a-11e9-8826-cc11449e1373.png)

**Comment by nul800sebastiaan** [comment]
We'll group the similar feature requests here, so from @jameswilki : 

> The Environment history view displays the timestamp from the commit. Please can the datetime of deployment be added as well, and the commits grouped under it so that we can see when code was deployed to an environment?

**Comment by sajumb** [comment]
Thanks @hfloyd for this feature request. 
Better late, than never as they say. We have added this to the backlog and are likely to consider it in one of the next sprints.
